opentreemap/importer/templates/importer/partials/resolve_popover.html

```diff
@@ -21,3 +21,9 @@
 data-url="{% url 'importer:update_row' instance_url_name=request.instance.url_name import_type=panel.import_type row_id=field.row_id %}"
 data-field-name="{{ field.name }}"
 {% endblock extra_save_attributes %}
+
+{% block suggestion %}
+{% if field.suggestion %}
+<p class="suggestion">{{ field.suggestion }}</p>
```

Comment:
I have no problem with this, but I'm raising it as a question. I feel like we don't have a strong convention here, but I feel like we often use the pattern of leaving empty dom elements for empty values, taking advantage of silent failing of the template syntax, and configuring the CSS to avoid any spacing when the element is empty.

In this case, that would just mean removing the `if` clause entirely.

Does that sound like the more common approach? Is that a pattern worth considering a convention, and using here instead?